client/my-sites/media/main.jsx

```diff
@@ -92,14 +95,22 @@ class Media extends Component {
 			currentDetail: null,
 			selectedItems: [],
 		} );
+		this.singleMediaViewRedirectToAllMedia();
+	};
+
+	singleMediaViewRedirectToAllMedia = () => {
```

Comment:
Oh, that's a long name - I'd name that one `maybeRedirectToAll` or something - but don't feel obligated to change it if you prefer this one.